Branch-TestBed/Branch-TestBed/ViewController.m

```diff
@@ -52,17 +53,25 @@ - (IBAction)cmdRefreshShort:(id)sender {
 - (IBAction)cmdShareLink:(id)sender {
     BranchLinkProperties *linkProperties = [[BranchLinkProperties alloc] init];
     linkProperties.feature = @"sharing";
+    linkProperties.alias = @"some alias";
+    linkProperties.tags = @[@"tag1", @"tag2"];
+    linkProperties.matchDuration = 123;
```

Comment:
nit: but can you make this 6000 or something so people don't think matching is broken while testing with our testbed.